src/rtpghi~.c

```diff
@@ -194,15 +293,25 @@ void rtpghi_tilde_setup(void) {
                                A_GIMME,
                               0);
 	//to do: maybe change tol_pd on runtime (while DSP turned ON)
-	/*class_addbang  (rtpghi_tilde_class, rtpghi_tilde_res);*/
+	//class_addbang  (rtpghi_tilde_class, rtpghi_tilde_reset);
 	class_addmethod(rtpghi_tilde_class,
 			(t_method)rtpghi_tilde_causal, gensym("causal"),
 			A_DEFFLOAT, 0);
 	
 	class_addmethod(rtpghi_tilde_class,
 			(t_method)rtpghi_tilde_tol, gensym("tolerance"),
 			A_DEFFLOAT, 0);
+	class_addmethod(rtpghi_tilde_class,
+			(t_method)rtpghi_tilde_overlap, gensym("overlap"),
+			A_DEFFLOAT, 0);
+	
+	class_addmethod(rtpghi_tilde_class,
+			(t_method)rtpghi_tilde_window, gensym("window"),
+			A_DEFSYMBOL, 0);
+	
 	class_addmethod(rtpghi_tilde_class,
 				   (t_method)rtpghi_tilde_dsp, gensym("dsp"), A_CANT, 0);
 	CLASS_MAINSIGNALIN(rtpghi_tilde_class, t_rtpghi_tilde, f);
+	class_sethelpsymbol(rtpghi_tilde_class, gensym("rtpghi~"));
```

Comment:
useless. avoid using `class_sethelpsymbol` as much as possible (it's really only useful for multiple objects sharing the same help-patch (which should be avoided), and objects with names that are not legal filenames (e.g. `///`))